x-pack/plugin/identity-provider/src/main/java/org/elasticsearch/xpack/idp/saml/sp/ServiceProviderCacheSettings.java

```diff
@@ -26,7 +26,7 @@
     public static final Setting<TimeValue> CACHE_TTL
         = Setting.timeSetting("xpack.idp.sp.cache.ttl", CACHE_TTL_DEFAULT, Setting.Property.NodeScope);
 
-    static <K, V> Cache<K, V> buildCache(Settings settings) {
+    public static <K, V> Cache<K, V> buildCache(Settings settings) {
```

Comment:
why?

Reply:
I was going to re-use this for the application privileges cache, but decided against it.